packages/taquito-rpc/test/data/rpc-responses.ts

```diff
@@ -4777,6 +4777,454 @@ export const blockKathmandunetSample = {
+    "test_chain_status": {
+      "status": "not_running"
+    },
+    "max_operations_ttl": 120,
+    "max_operation_data_length": 32768,
+    "max_block_header_length": 289,
+    "max_operation_list_length": [
+      {
+        "max_size": 4194304,
+        "max_op": 2048
+      },
+      {
+        "max_size": 32768
+      },
+      {
+        "max_size": 135168,
+        "max_op": 132
+      },
+      {
+        "max_size": 524288
+      }
+    ],
+    "proposer": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+    "baker": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+    "level_info": {
+      "level": 8215,
+      "level_position": 8214,
+      "cycle": 64,
+      "cycle_position": 22,
+      "expected_commitment": false
+    },
+    "voting_period_info": {
+      "voting_period": {
+        "index": 64,
+        "kind": "proposal",
+        "start_position": 8192
+      },
+      "position": 22,
+      "remaining": 105
+    },
+    "nonce_hash": null,
+    "deactivated": [],
+    "balance_updates": [
+      {
+        "kind": "accumulator",
+        "category": "block fees",
+        "change": "-1066",
+        "origin": "block"
+      },
+      {
+        "kind": "minted",
+        "category": "baking rewards",
+        "change": "-10000000",
+        "origin": "block"
+      },
+      {
+        "kind": "contract",
+        "contract": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+        "change": "10001066",
+        "origin": "block"
+      },
+      {
+        "kind": "minted",
+        "category": "baking bonuses",
+        "change": "-9999238",
+        "origin": "block"
+      },
+      {
+        "kind": "contract",
+        "contract": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+        "change": "9999238",
+        "origin": "block"
+      }
+    ],
+    "liquidity_baking_toggle_ema": 0,
+    "implicit_operations_results": [
+      {
+        "kind": "transaction",
+        "storage": [
+          {
+            "int": "1"
+          },
+          {
+            "int": "20535000100"
+          },
+          {
+            "int": "100"
+          },
+          {
+            "bytes": "01e927f00ef734dfc85919635e9afc9166c83ef9fc00"
+          },
+          {
+            "bytes": "0115eb0104481a6d7921160bc982c5e0a561cd8a3a00"
+          }
+        ],
+        "balance_updates": [
+          {
+            "kind": "minted",
+            "category": "subsidy",
+            "change": "-2500000",
+            "origin": "subsidy"
+          },
+          {
+            "kind": "contract",
+            "contract": "KT1TxqZ8QtKvLu3V3JH7Gx58n7Co8pgtpQU5",
+            "change": "2500000",
+            "origin": "subsidy"
+          }
+        ],
+        "consumed_milligas": "204995",
+        "storage_size": "4632"
+      }
+    ],
+    "proposer_consensus_key": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+    "baker_consensus_key": "tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE",
+    "consumed_milligas": "6009000"
+  },
+  "operations": [
+    [
```

Comment:
Consider deleting some parts of the block response when they are not relevant to the test.

Reply:
🙏